packages/DataGrid/src/DataGrid.Rows.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/prop-types */
```

Comment:
I extracted all the four grid's rows from the DataGrid: `HeaderSticky, Header, StickyGrid and MainGrid` into this file **DataGrid.Rows.js**.

The idea is to apply React.memo as it is suggested on React-window library to improve the performance of the DataGrid [info](https://react-window.now.sh/#/examples/list/memoized-list-items); also the use of `memonize-one` is required to memonize the data passdown to the React.memo components.